nni/experiment/launcher.py

```diff
@@ -109,15 +109,23 @@ def start_experiment(action, exp_id, config, port, debug, run_mode, url_prefix):
         _logger.info('Setting up...')
         rest.post(port, '/experiment', config.json(), url_prefix)
 
-        return proc
-
     except Exception as e:
         _logger.error('Create experiment failed')
         if proc is not None:
             with contextlib.suppress(Exception):
                 proc.kill()
         raise e
 
+    link = Path(config.experiment_working_directory, 'latest')
```

Comment:
Suggest `.latest` so that it's not flooded by tons of folders in `nni-experiments`, and people can find it with a first glance of `ls -a`.